core/base/src/TROOT.cxx

```diff
@@ -2737,6 +2737,31 @@ void TROOT::SetMacroPath(const char *newpath)
       macroPath = newpath;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// If `web` is not empty it defines where web graphics should be rendered
+///
+/// - "cef" web graphics is rendered via the Chromium Embedded Framework
+/// - "qt5" web graphics is rendered via Qt5
+/// - "browser" or "" web graphics is rendered in the default web browser
+///
+/// It also turn the normal graphics to Batch to avoid the loading of local
+/// graphics libraries.
+
+void TROOT::SetWeb(TString web)
+{
+   fWeb = web;
+   fWeb.ToLower();
+   if (fWeb.Contains("cef") || fWeb.Contains("qt5") || fWeb.Contains("browser") || fWeb.Length() ==0) {
+      fIsWeb = 1;
+      if (gROOT->IsBatch()) fIsWeb++;
+      gROOT->SetBatch(kTRUE);
+   } else {
+      fWeb = "";
+      fIsWeb = 0;
+      Warning("SetWeb", "Invalid web display");
```

Comment:
Disabling of web display should be a normal operation - there should be no warning

Reply:
well, it is mainly to tell warn people about a misspelled value... Like:

root  --web qr5

will tell the user that qr5 is in unknown option.